project-base/app/webpack.config.js

```diff
@@ -27,6 +27,10 @@ Encore
     .configureBabel(null, {
         includeNodeModules: ['@shopsys'],
     })
+    .addRule({
+        test: /\.svg/,
+        type: 'asset/source'
+    })
```

Comment:
Shouldn't we remove ignore line few lines below? 

```diff
{
                from: 'assets/public',
                to: '../../web/public',
                globOptions: {
-                    ignore: ['assets/public/admin/svg/**/*']
                },
                force: true
            },
```

Reply:
fixed in 3709373b86